content/reference-amendments.md

```diff
@@ -5,13 +5,13 @@ The following is a comprehensive list of all known amendments and their status o
 
 | Name                      | Introduced | Status                              |
 |:--------------------------|:-----------|:------------------------------------|
-| [Checks][]                | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [DepositAuth][]           | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [FlowCross][]             | v0.70.0    | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [SHAMapV2][]              | v0.33.0    | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [CryptoConditionsSuite][] | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [OwnerPaysFee][]          | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY")  |
-| [Tickets][]               | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY")  |
+| [FlowCross][]             | v0.70.0    | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [SHAMapV2][]              | v0.33.0    | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [Checks][]                | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [DepositAuth][]           | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [CryptoConditionsSuite][] | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [OwnerPaysFee][]          | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY") |
+| [Tickets][]               | TBD        | [Planned: TBD]( "BADGE_LIGHTGREY") |
```

Comment:
I may be misunderstanding the sort order - but should CryptoConditionsSuite come before DepositAuth?

Reply:
Oh, I think I know what's going on here. The sorting criteria within "Planned" amendments is the "Introduced" version (in descending order) but that means that ones with the same "TBD" end up in non-deterministic order.

I can fix that by adding another level of sorting based on the name.